ops-bedrock/docker-compose.yml

```diff
@@ -172,6 +217,9 @@ services:
       OP_BATCHER_BATCH_TYPE: 0
       OP_BATCHER_PLASMA_ENABLED: "${PLASMA_ENABLED}"
       OP_BATCHER_PLASMA_DA_SERVER: "http://da-server:3100"
+      # uncomment to use blobs
+      # (requires L1 Dencun and L2 Ecotone activation first)
+      # OP_BATCHER_DATA_AVAILABILITY_TYPE: blobs
```

Comment:
Seems like we should not have to make a diff to use this but instead use env var templating, similar to this line:

```
      OP_CHALLENGER_GAME_FACTORY_ADDRESS: ${DGF_ADDRESS}
```

so we could do something like:

```
      OP_BATCHER_DATA_AVAILABILITY_TYPE: ${OP_BATCHER_DATA_AVAILABILITY_TYPE:-calldata}
```

Reply:
Good idea, I will do this in the follow up to this PR where I enable L1 Cancun at genesis in the devnet. Right now, this cannot be set to blobs because L1 Cancun only activates after 0x30 seconds.